src/midiEditorCommands.cpp

```diff
@@ -492,12 +510,24 @@ void cmdMidiInsertNote(Command* command) {
 	MIDIEditor_OnCommand(editor, command->gaccel.accel.cmd);
 	if (MIDI_CountEvts(take, NULL, NULL, NULL) <= oldCount)
 		return; // Not inserted.
-	// Play the inserted note.
 	int pitch = MIDIEditor_GetSetting_int(editor, "active_note_row");
-	int chan = MIDIEditor_GetSetting_int(editor, "default_note_chan");
-	int vel = MIDIEditor_GetSetting_int(editor, "default_note_vel");
-	previewNotes(take, {{chan, pitch, vel}});
-	outputMessage(formatCursorPosition(TF_MEASURE));
+	// Get selected notes.
+	vector<MidiNote> selectedNotes = getSelectedNotes(take);
+	// Find the just inserted note based on its pitch, as that makes it unique.
+	auto note = *(find_if(
+		selectedNotes.begin(), selectedNotes.end(),
+		[pitch](MidiNote n) { return n.pitch == pitch; })
+	);
+	// Play the inserted note.
+	previewNotes(take, {note});
+	ostringstream s;
+		if (shouldReportNotes) {
```

Comment:
nit: Extraneous tab.
```suggestion
	if (shouldReportNotes) {
```